csharp-basic-topics/AwaitInForEachLoopCsharp/AwaitInForEachLoopCsharp/TaskWhenAllInForEachLoop.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+
+namespace AwaitInForEachLoop
+{
+    public class TaskWhenAllInLoop
+    {
+        public static async Task<List<int>> ResultAsync()
+        {
+            var numbers = new List<int> { 10, 20, 30, 40, 50, 60, 70, 80, 90 };
+
+            var processingTasks = new List<Task>();
+
+            foreach (int number in numbers)
+            {
+                PrepNumbers();
```

Comment:
Remove this list also and the function

Reply:
You want me to remove this list
 var numbers = new List<int> { 10, 20, 30, 40, 50, 60, 70, 80, 90 };
 and this function
 PrepNumbers();
 
 What will be a better way to go about it

Reply:
Sorry for the confusion here, I mean remove PrepNumbers() function call and the function, if not needed.